### PR TITLE
Align insights preview with investigation service

### PR DESCRIPTION
**Pull request by rpressburger**
There were some gaps between the investigation service and github.
We align both.

### CODE REVIEW

**Review by aprakash13 — CHANGES_REQUESTED** [review]
Made a few minor comments that probably need to be looked at before we can go ahead and merge these. Thanks, Royi.

File: Insights/Account/AzActivity_AzureOps.yaml [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
   let upn = strcat(Account_Name,"@",Account_UPNSuffix);
-  AzureActivity 
+  AzureActivity
   | where (isnotempty(Account_AadUserId) and Caller =~ Account_AadUserId) or Caller =~ upn
-  | where OperationName contains "Run Command on Virtual Machine"
-               or (OperationName == "List Storage Account Keys" and ActivityStatus == "Succeeded")
-               or OperationName == "Create or Update Virtual Machine" 
-               or OperationName == "Create Deployment"
-               or OperationName == "Create role assignment"
+  | where OperationNameValue contains "Run Command on Virtual Machine"
```
[the remark this answers is not in the document]
Comment (rpressburger) [reply Insights/Account/AzActivity_AzureOps.yaml]: Value updated

**Comment by rpressburger** [comment]
@aprakash13 Can you please take a look again?

**Review by aprakash13 — APPROVED** [review]
Thanks for the changes @rpressburger.

**Pull request merged**